client/stylesheets/_toCat.scss

```diff
@@ -5316,6 +5336,7 @@ On click the images will be displayed at normal size to complete the effect
 .optionsListClick ul {
 	position:relative;
 	font-weight:400;
+    margin: 5px 0 0 0 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `margin` should be written more concisely as `5px 0 0` instead of `5px 0 0 0`